lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+  attr_reader :dishes
+
+  def initialize
+  @dishes = {
+    Pizza: 10,
+      Burger: 5,
+      Fish: 1
+    }
+end
+
+def print
+  @dishes.map do |dish, price|
+    "%s £%.2f" % [dish.to_s.capitalize, price]
```

Comment:
is this regex?

Reply:
i dont know... found this code on another page...it works